opslevel/resource_opslevel_check_repository_search.go

```diff
@@ -70,7 +70,9 @@ func NewCheckRepositorySearchResourceModel(ctx context.Context, check opslevel.C
 
 	data, diags := types.ListValueFrom(ctx, types.StringType, check.RepositorySearchCheckFragment.FileExtensions)
 	stateModel.FileExtensions = data
-	stateModel.FileContentsPredicate = NewPredicateModel(check.RepositorySearchCheckFragment.FileContentsPredicate)
+	if check.RepositorySearchCheckFragment.FileContentsPredicate != nil {
```

Comment:
This change is dependent on an [opslevel-go PR](https://github.com/OpsLevel/opslevel-go/pull/406)

Reply:
Merged ✅